src/misc/lv_fs.h

```diff
@@ -110,6 +110,12 @@ typedef struct {
     lv_fs_file_cache_t * cache;
 } lv_fs_file_t;
 
+typedef struct {
+    char            path[4];   // always "M:x"
```

Comment:
Why 4 bytes and not 3? 
```suggestion
    char            path[3];   /* always "M:" */
```

Reply:
For dword alignment of the other fields. Probably the compiler would add one byte automatically anyway, but this is clearer.